lib/util/SortableSet.js

```diff
@@ -94,4 +92,16 @@ module.exports = class SortableSet extends Set {
 		this._cacheOrderIndependent.set(fn, newData);
 		return newData;
 	}
-};
+
+	_invalidateCache(all) {
+		if(this._cache !== null)
+			this._cache.clear();
+		if(all && this._cacheOrderIndependent !== null)
+			this._cacheOrderIndependent.clear();
+	}
+}
+
+SortableSet.prototype._cache = null;
+SortableSet.prototype._cacheOrderIndependent = null;
```

Comment:
These statements are required because `_invalidateCache` is called by `add` during `super()` call.